libs/src/oracle-sdk-v2/services/oraclev3/gql/queries.ts

```diff
@@ -1,33 +1,13 @@
 import { chainsById } from "@shared/constants";
-import type {
-  ChainId,
-  ErrorMessage,
-  OOV3GraphEntity,
-  OOV3GraphQuery,
-} from "@shared/types";
+import type { ChainId, OOV3GraphEntity, OOV3GraphQuery } from "@shared/types";
 import { makeQueryName } from "@shared/utils";
 import request, { gql } from "graphql-request";
 
-export async function getAssertions(
-  url: string,
-  chainId: ChainId,
-  addErrorMessage: (message: ErrorMessage) => void
-) {
-  try {
-    const chainName = chainsById[chainId];
-    const queryName = makeQueryName("Optimistic Oracle V3", chainName);
-    const result = await fetchAllAssertions(url, queryName);
-    return result;
-  } catch (e) {
-    addErrorMessage({
```

Comment:
again this is removed to prefer emitting this in the oracle context. we need to be able to check errors there in order to fallback to correct web3 service